### ISSUE TITLE
Not Working in FireFox

### ISSUE THREAD

**Issue by HammadMehar** [VincentCharpentier/Simple-MonthPicker#6]
Month Selection not working in FireFox when use with bootstrap popup model

**Comment by VincentCharpentier** [comment]
can you provide a minimal code to reproduce please ?
Also, which version of bootstrap are you using ?

**Comment by HammadMehar** [comment]
Bootstrap 3.3.5
Here is the CODE
*Partial View*

<div class="modal-dialog">
    <div class="modal-content">

        @using (Ajax.BeginForm("AddDurationPartial", "Portfolio", null, new AjaxOptions { UpdateTargetId = "divDurationIndex", InsertionMode = InsertionMode.Replace, HttpMethod = "Post", OnSuccess = "budgetAssetsDuration.onCreateDurationSuccess", OnFailure = "UpdatePayError" },
                new { @id = "Addurationform" }))
        {
            @Html.HiddenFor(x => x.company_key)
            @Html.ValidationSummary(true)
            <div class="modal-header bg-theme">
                <button type="button" class="close" onclick="budgetAssetsDuration.cancelDurationPopup()" aria-hidden="true"></button>
                <h4 class="modal-title" id="DurationTitle"> Add Group </h4>
            </div>
            <div class="modal-body">
                <div class="portlet-body form">
                    <div class="row">
                        <div class="col-lg-12 form-group">
                            <div class="col-lg-3">
                                <label class="control-label">From</label>
                            </div>
                            <div class="col-lg-9">
                                @Html.TextBoxFor(x => x.StartDate, new { @class = "form-control startDateDuration" })
                                @Html.ValidationMessageFor(x => x.StartDate)
                                @Html.HiddenFor(x => x.StartDate, new { @id = "hdDurationStart2" })
                                @Html.HiddenFor(x => x.Id)
                                @Html.HiddenFor(x => x.company_key, new { @id = "company_key" })
                            </div>
                        </div>
                        <div class="col-lg-12 form-group">
                            <div class="col-lg-3">
                                <label class="control-label">To</label>
                            </div>
                            <div class="col-lg-9">
                                @Html.TextBoxFor(x => x.Enddate, new { @class = "form-control endDateDuration" })
                                @Html.ValidationMessageFor(x => x.Enddate)
                                @Html.HiddenFor(x => x.Enddate, new { @id = "hdDurationEnd2" })
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        }
        <div class="modal-footer">
            <div class="col-md-12">
                <button type="button" id="btnSaveDuration" onclick="budgetAssetsDuration.onAddDuration();" class="btn btn-default">Save</button>
                <button type="button" class="btn grey-mint btn-outline" onclick="budgetAssetsDuration.cancelDurationPopup()">Cancel</button>
            </div>
        </div>
    </div>
</div>

*Initialization*
        $('.startDateDuration').Monthpicker({
            onSelect: function () {
                debugger
                $('.endDateDuration').Monthpicker('option', { minValue: $('.startDateDuration').val() });
                //$("#Addurationform").setTrackChange();
            }
        });
        $('.endDateDuration').Monthpicker({
            onSelect: function () {
                debugger
                $('.startDateDuration').Monthpicker('option', { maxValue: $('.endDateDuration').val() });
            }
        });
Here Is the Video Link of Issue

https://www.screencast.com/t/D8xqjsEQ
